### DISCUSSION TITLE
#复现不出来预训练模型的精度

### DISCUSSION THREAD

**Discussion by xumengmeng-97** [STVIR/pysot#573] [Code]
I set siamrpn_mobilev2_dwxcorr.yaml to train.py directly, using 4 2080Ti graphics cards, num_worker is 16, the accuracy of reproduction is very different, does anyone know the reason, please hel
![捕获](https://user-images.githubusercontent.com/68225340/169840601-c4d678b5-2642-4e28-9e1c-419e373af15f.PNG)
p

**Comment by sssimpleboy** [comment]
你的准确率怎么看 的？ 我为什么训练时候只能看到损失呀
